commit/tokenprice/processor.go

```diff
@@ -47,6 +87,97 @@ func (w *Processor) ValidateObservation(
 	return nil
 }
 
+func (p *Processor) ObserveFDestChain() (*int, error) {
```

Comment:
This is a really wacky function signature, why return a pointer to an int?

Reply:
Yeah, return 0. If an error is returned, the value should never be used as its undefined.